modules/installation-configuration-parameters.adoc

```diff
@@ -1070,7 +1070,7 @@ Additional Azure configuration parameters are described in the following table:
 |String, for example `production_disk_encryption_set`.
 
 |`compute.platform.azure.osDisk.diskEncryptionSet.subscriptionId`
-|Optional. The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt compute machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
+|The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt compute machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
```

Comment:
Installer will not set default subscriptionId any more if use doesn't provide it. This field is required one and need user provide. So below statements should be removed.
`By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.`